sdk/formrecognizer/ci.yml

```diff
@@ -27,6 +27,7 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: formrecognizer
+    SpellCheckPublicApiSurface: true
```

Comment:
Why only turn this on only for this pipeline?

Reply:
@maririos and @heaths want to roll this out iteratively so we're leaving it off by default and allowing services to opt-in. Eventually we'll be able to flip the default to `true` and make the checking opt-out. 

By way of example, the Form Recognizer fails with a false positive on a language code that needed to be added to an exception: https://github.com/Azure/azure-sdk-for-net/pull/20088/files#diff-d9f1a54e1f1a4c8324f4756f08a6af27671a3ae9312190d4a5fe981cc1722824R27

We don't want to suddenly start blocking everyone.

Reply:
OK I guess I didn't expect us to have many hits in our apis at this point but if that is the plan then I'm cool with that. Lets make sure we have issues tracking getting clean and flipping this default.

Reply:
There's quite a few, and all of them I've seen in my APIs are false negatives. Seems there's some fairly obvious tech words not in their dictionary we just need to add. To their credit, though, there's quite a few that are recognized that you won't find in a "normal" dictionary.